src/main/java/ColorsSupplier.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Random;
+public class ColorsSupplier {
+    private static final Random RANDOM = new Random();
+    public static Colors getRandomColor(){
+        Colors[] someColor = Colors.values();
+        int index = new Random().nextInt(Colors.values().length);

```

Comment:
Consider removing the extra `new Random()` instantiation and use the static `RANDOM` field instead. 💡